apps/web/app/(admin)/admin/projects/page.tsx

```diff
@@ -149,10 +159,14 @@ export default function AdminProjectsDashboard() {
         (project.gitRepoUrl?.toLowerCase().includes(searchLower) ?? false)
       );
     })
-    .filter((project) => statusFilter === 'all' || project.status === statusFilter)
-    .filter((project) => typeFilter === 'all' || project.type === typeFilter)
+    .filter((project: Project) => statusFilter === 'all' || project.status?.name === statusFilter)
```

Comment:
```suggestion
    .filter((project) => statusFilter === 'all' || project.status?.name === statusFilter)
```

no need for that, typescript already infers it